doc/rst/source/supplements/seis/seis_extra_cols.rst_

```diff
@@ -0,0 +1,16 @@
+Data Column Order
+-----------------
+
+The **-S** option determines how many data columns are required to generate
+the selected symbol.  In addition, your use of options **-I** and **-t** will
+require extra columns, as will a **-S** option without the *scale*.  The
+order of the data record is fixed, even if not all items may be activated.
+We expect data columns to come in the following order::
+
+    lon lat symbol-columns [scale] [intens] [transp [transp2]] [trailing-text]
```

Comment:
`depth` is always required in **meca** and **coupe**, unless **-Fo** is used
```suggestion
    lon lat depth symbol-columns [scale] [intens] [transp [transp2]] [trailing-text]
```

Reply:
Perhaps place depth in brackets and point out it is only excluded when -Fo is used?

Reply:
OK to me.